options/locale/locale_sv-SE.ini

```diff
@@ -392,8 +392,8 @@ manage_gpg_keys=Hantera GPG-nycklar
 add_key=Lägg till nyckel
 ssh_desc=Dessa publika SSH nycklar är associerade med ditt konto. De motsvarande privata nycklarna tillåter full åtkomst till dina utvecklingskataloger.
 gpg_desc=Dessa publika GPG nycklar är associerade med ditt konto. Håll dina privata nycklar säkra då de tillåter att commits kan verifieras.
-ssh_helper=<strong>Behöver du hjälp?</strong> Kolla in Github's guide för att <a href="%s">skapa din egen SSH-nycklar</a> eller lösa <a href="%s">vanliga problem</a> som kan uppstå med SSH.
-gpg_helper=<strong>Behöver du hjälp?</strong> Ta en titt på Github's guide <a href="%s"> om GPG</a>.
+ssh_helper=<strong>Behöver du hjälp?</strong> Kolla in GitHub's guide för att <a href="%s">skapa din egen SSH-nycklar</a> eller lösa <a href="%s">vanliga problem</a> som kan uppstå med SSH.
+gpg_helper=<strong>Behöver du hjälp?</strong> Ta en titt på GitHub's guide <a href="%s"> om GPG</a>.
```

Comment:
The CI will overwrite any changes made to the locales that aren't en-US. To change the translations please use: https://crowdin.com/project/gitea